staging/src/k8s.io/client-go/util/client/clientutil.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package keyutil contains utilities for managing public/private key pairs.
+package clientutil
+
+import (
+	"fmt"
+	clientset "k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func GetClientFromFile(kubeconfigPath string) (clientset.Interface, error) {
+
+	clientConfig, err := CreateClientConfigFromFile(kubeconfigPath)
+	if err != nil {
+		return nil, err
+	}
+
+	client, err := clientset.NewForConfig(clientConfig)
+	if err != nil {
+		return nil, fmt.Errorf("error while creating clientset with %s, error %v", kubeconfigPath, err.Error())
+	}
+
+	return client, nil
+}
+
+func CreateClientConfigFromFile(kubeconfigPath string) (*restclient.Config, error) {
+	return CreateClientConfigFromSpecificFile(kubeconfigPath, 0, 0, "")
+}
+
+func CreateClientConfigFromSpecificFile(kubeconfigPath string, qps float32, burst int, contentType string) (*restclient.Config, error) {
```

Comment:
Name is a bit confusion. Perhaps rename to CreateClientConfigFromKubeConfigFile. Similar to above util functions

Reply:
i agree ( i was a bit on this as well when i got the function name ). will make a change to it.

Reply:
fixed